apps/wallet/src/ui/app/components/accounts/AccountBalanceItem.tsx

```diff
@@ -90,6 +91,14 @@ export function AccountBalanceItem({
         OBJECT_PER_REQ,
     );
 
+    const hasVestingObjectsWithLabel = (vestingObjects?.pages?.[0]?.[0]?.data || []).some(
```

Comment:
we could have the supply increase vesting object on another page other than the first one.
I think we should check if the first page has one of supply increase vesting object, and if not, keep fetching pages until there are no more